### DISCUSSION TITLE
Thanks @ntimo

### DISCUSSION THREAD

**Discussion by bpmct** [coder/coder#3360] [Announcements]
Wanted to give a second to give @ntimo a huge thanks for their contributions to the `coder/coder` 🎉

Timo was probably our first `coder/coder` user and has been sharing [valuable feedback](https://github.com/coder/coder/issues?q=author%3Antimo+) and [contributions](https://github.com/coder/coder/pulls?q=+author%3Antimo+) to the project, such as the original Docker template. (Also an [unofficial Hetzner template](https://github.com/ntimo/coder-hetzner-cloud-template))

From myself and on behalf of the Coder team, thank you 👨🏼‍💻 

--

Inspired by @jsjoeio's [code-server community shoutouts](https://github.com/coder/code-server/discussions/5365)

**Comment by kylecarbs** [comment]
Thank you @ntimo 🥳

**Reply by ntimo** [reply]
Thank you @kylecarbs and @bpmct its been truly a lot of fun using coder and building templates using tf. Its probably my most used software besides chrome/firefox. I would also like to thank you and the team for building such an awesome piece of tech.
